adafruit_wiznet5k/adafruit_wiznet5k.py

```diff
@@ -61,6 +62,10 @@
 _REG_SIPR = const(0x000F)  # Source IP Address
 _REG_PHYCFGR = const(0x002E)  # W5500 PHY Configuration
 _REG_PHYCFGR_W5100S = const(0x003C)  # W5100S PHY Configuration
+_REG_RCR_5100s = const(0x0019)  # Retry Count
```

Comment:
Q: You removed `dhcp_timeout` from the class initialization method in favor of hardcoding these values. Will users _always_ want to use this retry time/retry count, or is there a future need for it to be set-able by a user?

Reply:
The `dhcp_timeout` set by the user was used to decide how long to wait for a response to a DHCP request from the server. This is obsolete as the time out now follows the DHCP standard of  2 +/- 1 seconds, 4 +/- 1 seconds, etc up to 32 +/- 1 seconds. The new retry time and count is used to implement that standard. I don't think this is something the user needs to adjust, but I'm happy to discuss possible use cases.

Reply:
> This is obsolete as the time out now follows the DHCP standard

Sweet, thank you for the clarification.